MAUI/TabView/Tab-Item-Customization.md

```diff
@@ -478,4 +478,22 @@ tabView.IsScrollButtonEnabled = true;
 
 {% endtabs %}
 
-![TabView Scroll Mode](images/TabViewScroll.gif) 
\ No newline at end of file
+![TabView Scroll Mode](images/TabViewScroll.gif) 
+
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```

Comment:
The `FontAutoScalingEnabled` property is used to automatically scale the tab header's font size based on the operating system's text size. The default value of the `FontAutoSaclingEnabled` property is `false.`

Reply:
Yes, changed.